File: android/src/main/java/com/Mindelo/Ventoura/Entity/City.java
package com.Mindelo.Ventoura.Entity;

import java.io.Serializable;
import lombok.Data;

@Data
public class City implements Serializable {
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;

	private int id;

	private String cityName;
	private int countryId;
	
	private String countryName;
}

File: 1-other-artifacts/ios/Ventoura/Model/City.m
//
//  City.m
//  Ventoura
//
//  Created by Wenchao Chen on 15/08/2014.
//  Copyright (c) 2014 Ventoura. All rights reserved.
//

#import "City.h"

@implementation City

- (instancetype)initWithId:(NSString *)cityId cityName:(NSString *)cityName countryId:(NSString *)countryId countryName:(NSString *)countryName tableIndexPath:(NSUInteger)tableIndexPath{
    self = [super init];
    if (self) {
        _cityId = cityId;
        _countryId= countryId;
        _countryName = countryName;
        _tableIndexPath = tableIndexPath;
        _cityName = cityName;
    }
    return self;
}
@end
